.claude/rules/testing.md

```
@@ -0,0 +1,89 @@
+# Testing
+
+## Current state
+
+There are no test projects in the repository. The CI workflow (`dotnet test`) will no-op until test projects are added.
+
+## Recommended project structure (when adding tests)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)